gdk/commands/test/RunCommand.py

```diff
@@ -34,18 +35,44 @@ def run(self):
             logging.info("Downloading latest nucleus archive from url %s", self._nucleus_archive_link)
             URLDownloader(self._nucleus_archive_link).download(_nucleus_path)
 
+        self._run_testing_jar()
+
     def _is_test_module_built(self) -> bool:
         """
         Return true if the test module build folder exists
         """
-        uat_build_system = UATBuildSystem.get(self._test_build_system)
-        test_build_folder = self._test_directory.joinpath(*uat_build_system.build_folder).resolve()
+        return self._test_build_directory().exists()
 
-        return test_build_folder.exists()
+    def _test_build_directory(self) -> Path:
+        """
+        Return the test build directory
+        """
+        uat_build_system = UATBuildSystem.get(self._test_build_system)
+        return self._test_directory.joinpath(*uat_build_system.build_folder).resolve()
 
     def _should_download_nucleus_archive(self, _nucleus_path: Path) -> bool:
         """
         If ggc-archive path is set to default path and if it doesn't already exist at this path, then download the latest
         nucleus archive from url.
         """
         return _nucleus_path == Path(self._config.default_nucleus_archive_path).resolve() and not _nucleus_path.exists()
+
+    def _run_testing_jar(self) -> bool:
```

Comment:
```suggestion
    def _run_testing_jar(self) -> None:
```
 or will return something eventually?